virtual-shared/integration/src/layers.ts

```diff
@@ -3,19 +3,17 @@ export const VIRTUAL_ENTRY_ALIAS = [
 ]
 export const LAYER_MARK_ALL = '__ALL__'
 
-export const RESOLVED_ID_WITH_QUERY_RE = /[/\\]__uno(_.*?)?\.css(\?.*)?$/
-export const RESOLVED_ID_RE = /[/\\]__uno(?:_(.*?))?\.css$/
+export const RESOLVED_ID_WITH_QUERY_RE = /\0[/\\]__uno(_.*?)?\.css(\?.*)?$/
+export const RESOLVED_ID_RE = /\0[/\\]__uno(?:_(.*?))?\.css$/
 
 export function resolveId(id: string) {
   if (id.match(RESOLVED_ID_WITH_QUERY_RE))
-    return id
+    return `\0${id}`
 
   for (const alias of VIRTUAL_ENTRY_ALIAS) {
     const match = id.match(alias)
     if (match) {
-      return match[1]
-        ? `/__uno_${match[1]}.css`
-        : '/__uno.css'
+      return `\0${match[1] ? `/__uno_${match[1]}.css` : '/__uno.css'}`
```

Comment:
```suggestion
      return match[1]
        ? `\0/__uno_${match[1]}.css`
        : '\0/__uno.css'
```

I think this makes it easier to read